backend/core/src/migration/1657568202436-turn-settings-off.ts

```diff
@@ -0,0 +1,31 @@
+import { MigrationInterface, QueryRunner } from "typeorm"
+import { CountyCode } from "../shared/types/county-code"
+
+export class turnSettingsOff1657568202436 implements MigrationInterface {
```

Comment:
At the moment this feature is toggled by a jurisdictional setting `enablePartnersSettings`. It was turned on for some jurisdictions when the migration was added. Now that we are able to have different seed data than migration data due to Colin's recent changes, we can turn the flag off here but turn it on in the seeds to continue to have it default be on locally.